fsw/cfe-core/src/inc/cfe_sb_events.h

```diff
@@ -933,6 +933,19 @@
 **/
 #define CFE_SB_CR_PIPE_NO_FREE_EID      63
 
+/** \brief <tt> 'SB GetLastSender Err:CurrentBuff not initialized - No last sender, Pipe=%d,App=%s' </tt>
+**  \event <tt> 'SB GetLastSender Err:CurrentBuff not initialized - No last sender, Pipe=%d,App=%s' </tt>
+**
+**  \par Type: ERROR
+**
+**  \par Cause:
+**
+**  This error event message is issued when GetLastSender detects that the pntr to the buffer
+**  descriptor for the last msg received for a given pipe is NULL.  This usually occurs because
+**  CFE_SB_GetLastSenderId was called prior to recieving a msg.
+**/
+#define CFE_SB_NO_LAST_MESG_RECV_EID    64
```

Comment:
Does this really warrant an "error event"?  Seems like normal operational context - there is nothing inherently an "error" about a pipe that exists but hasn't been sent to yet.  

I can foresee this API being used to simply check if a pipe has been written to or not, where so long as the application handles the return code its not an error either way, and generating an "Error event" might be confusing.

Reply:
Just removed error event and pushed the updates.